dev-libs/libprelude/libprelude-4.0.0.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+USE_RUBY="ruby21 ruby22 ruby23 ruby24"
+
+inherit autotools python-r1 ruby-single
+
+DESCRIPTION="Prelude-SIEM Framework Library"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc lua python ruby perl"
+
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="dev-libs/libgcrypt:0=
```

Comment:
Do we still need libgcrypt? https://github.com/Prelude-SIEM/libprelude/compare/3.1.0...4.0.0#diff-ff4e2dc4962dc25a1512353299992c8dR23

Reply:
no indeed, good look !